File: touralbum/app/src/main/java/com/example/touralbum/ui/reminder/util/StringUtil.java
package com.example.touralbum.ui.reminder.util;

public class StringUtil {
    public static String getMultiNumber(int number) {
        return number < 10 ? "0" + number : Integer.toString(number);
    }

    public static String getLocalMonth(int month) {
        return getMultiNumber(month + 1);
    }

    public static boolean isBlank(String src) {
        return src == null || src.trim().length() == 0;
    }
}

File: touralbum/app/src/main/java/com/example/touralbum/ui/reminder/util/StringUtil.kt
package com.example.touralbum.ui.reminder.util

/**
 * @author jsbintask@gmail.com
 * @date 2018/4/26 15:00
 */
object StringUtil {
    fun getMultiNumber(number: Int): String {
        return if (number < 10) "0$number" else Integer.toString(number)
    }

    fun getLocalMonth(month: Int): String {
        return getMultiNumber(month + 1)
    }

    fun isBlank(src: String?): Boolean {
        return src == null || src.trim { it <= ' ' }.length == 0
    }
}